### DISCUSSION TITLE
In product feature notification patterns?

### DISCUSSION THREAD

**Discussion by mjanssen2021** [twilio-labs/paste#2189] [Q&A] [answered]
I am currently working on some updates for the Verify product - and we want to include in feature in app notifications to help users onboard better to two new product changes:

* A new page structure (tabs) that now have their settings organized per channel.
* A new feature (Geopermissions) in the console navigation. 

Do the patterns i'm presenting in these designs work okay for these use cases? Or do we have any other, more streamlined in app notification/education patterns I could use?

Thank you!

<img width="1450" alt="Verify (Look at your tabs!)" src="https://user-images.githubusercontent.com/78432794/152608364-10f0c3a9-e1ab-4aac-9ba5-01830770faf0.png">
<img width="872" alt="Screen Shot 2022-02-04 at 3 58 40 PM" src="https://user-images.githubusercontent.com/78432794/152608801-cc51092a-bb30-4f73-b185-5a0526ad31f8.png">

**Comment by gloriliale** [comment, marked as answer]
Hi @mjanssen2021! Thanks for sending these over! A few thoughts 💭 
1. New Tab Page Structure: we use this pattern in a few places, so it seems like it'd be intuitive from our point of view. Did you test it and users had trouble with the tabs?
2. New Geopermissions feature: what is it that you're thinking will trigger this popover? If it's the link itself, the click action is already being used to go to this page. Were you thinking of it appearing on page load/another trigger?

**Reply by mjanssen2021** [reply]
Hi there!

- It's been near impossible to find Verify users to test our designs on - but we just don't want customers to skip going through these tabs since it's new functionality for configuring their individual channel settings (currently the information architecture has these settings all unorganized on one page). So we wanted to call this out to users that this is a new pattern that is different from the old design and that they need to go to these tabs to configure the settings. 
- I was thinking that this would appear on page load.

**Reply by andioneto** [reply]
Hi @mjanssen2021 👋

I think we may need to get some input from the DX team since this page is within the Twilio Console .

@hharnisc What is the team using for this these days? One of our designers mentioned that [Appcues](https://www.appcues.com/) would be great for this, and that potentially is already part of the Console infrastructure. Thoughts?

**Reply by hharnisc** [reply]
Hey 👋 Appcues is the current tool we have for these types of flows. The tricky part is getting the help flows to show at the right time. Tooltips could potentially be an option as well. For some more help on the Console end this is a great question for #help-console-beta Slack channel.
